### DISCUSSION TITLE
How to get data?

### DISCUSSION THREAD

**Discussion by ddxv** [A-Lehmann-Elektro-AG/solar-flow-grafana#10] [General]
Where does the data come from? Do you collect it on your own local postgres database? Or is the idea to get it directly from a MPPT or do people add some kind of trackers on their inverters?

For reference, would love to add something like this for an older off-grid system with lead acid batteries. 

thanks!

**Comment by d1msk1y** [comment]
We personally use a local InfluxDB as a data source, where we have the data from an MPPT. However, you can use any data source available on Grafana to get your data. How you read your data, depends on the solar system manufacturer/provider. You can try contacting the support of your manufacturer or searching for help online

I'm not an expert, but I think the key is to have your data stored in a database.

How do you currently view your data?

**Reply by ddxv** [reply]
Currently it's not stored at all. There is a funky little wifi bridge hooked up to the old Midnite Classic 250. The wifi bridge works for a couple weeks before dying out, and sends the data to Midnite Solar or to an adobe windows app which is hard to get working on modern computers. Ultimately, its just not something my parents ever cared about, but something I love, so am just curious. I've been looking to buy them a new MPPT and recording/data collection was one of the things I was curious about.

Anyways, I guess the newer ones make data collection from the device much easier? I see they often now have bluetooth/wifi connectivity built in directly.

Thanks for your cool project, really exciting to see.

**Reply by d1msk1y** [reply]
Yeah, I think in your case, you do need to get an MPPT, otherwise, while not impossible, it is still tricky to get your system working with the plugin. 

Thanks :)
